### DISCUSSION TITLE
ScrollBar Hide

### DISCUSSION THREAD

**Discussion by mrnasil** [tailwindlabs/tailwindcss#6849] [Ideas]
Please added to hide scrollbars, although the element can still be scrolled if the element's content overflows.

**Comment by YusufcanY** [comment]
It will be really good. I using this plugin when i need to hide scrollbar: 
https://github.com/reslear/tailwind-scrollbar-hide

But why not, needs to be added.

**Comment by paragchirde** [comment]
```
::-webkit-scrollbar {
    width: 0px;
    background: transparent;
}
```

**Reply by YusufcanY** [reply]
For compatibility of browsers:
```
      /* IE and Edge */
      '-ms-overflow-style': 'none',

      /* Firefox */
      'scrollbar-width': 'none',

      /* Safari and Chrome */
      '&::-webkit-scrollbar': {
        display: 'none'
      }
```
